frontend/src/redux/actions-creators/user.ts

```diff
@@ -0,0 +1,23 @@
+import { Dispatch } from 'redux';
+import axios from 'axios';
+import { UserAction, UserActionTypes } from '../types/user';
+
+export const fetchUsers = () => async (dispatch: Dispatch<UserAction>) => {
+  try {
+    dispatch({ type: UserActionTypes.FETCH_USERS });
+    const response = await axios.get(
+      'https://jsonplaceholder.typicode.com/users'
+    );
+    setTimeout(() => {
+      dispatch({
+        type: UserActionTypes.FETCH_USERS_SUCCESS,
+        payload: response.data
+      });
+    }, 500);
+  } catch (e) {
+    dispatch({
+      type: UserActionTypes.FETCH_USERS_ERROR,
+      payload: 'Произошла ошибка при загрузке пользователей'

```

Comment:
maybe we need some distinct file with constants for all of the unchangeable (static) strings

Reply:
maybe we need to discuss tomorrow